player/lua/console.lua

```diff
@@ -1180,6 +1180,8 @@ local function property_list()
         properties[#properties + 1] = 'current-tracks/' .. sub_property
     end
 
+    properties[#properties + 1] = 'clipboard/text'
```

Comment:
```suggestion
    for _, sub_property in pairs({'text'}) do
        properties[#properties + 1] = 'clipboard/' .. sub_property
    end
```
Could we do this, so it is easier to extend to image and other formats in the future?